<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAChartCommonItemsSettingWidget</class>
 <widget class="QWidget" name="DAChartCommonItemsSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Common Item Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="DA::DAChartCurveItemSettingWidget" name="widgetCurveItem"/>
     <widget class="DA::DAChartBarItemSettingWidget" name="widgetBarItem"/>
     <widget class="DA::DAChartErrorBarItemSettingWidget" name="widgetErrorBarItem"/>
     <widget class="DA::DAChartLegendItemSettingWidget" name="widgetLegendItem"/>
     <widget class="DA::DAChartGridSettingWidget" name="widgetGridItem"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DAChartCurveItemSettingWidget</class>
   <extends>QWidget</extends>
   <header>DAChartCurveItemSettingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAChartGridSettingWidget</class>
   <extends>QWidget</extends>
   <header>DAChartGridSettingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAChartLegendItemSettingWidget</class>
   <extends>QWidget</extends>
   <header>DAChartLegendItemSettingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAChartErrorBarItemSettingWidget</class>
   <extends>QWidget</extends>
   <header>DAChartErrorBarItemSettingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAChartBarItemSettingWidget</class>
   <extends>QWidget</extends>
   <header>DAChartBarItemSettingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
